docs/content/hive-getting-started.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Hive and Iceberg Quickstart"
+weight: 100
+url: hive-quickstart
+aliases:
+    - "quickstart"
+    - "quickstarts"
+    - "getting-started"
+disableSidebar: true
+disableToc: true
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+<!-- {{% quickstarts %}} -->
+
+## Hive and Iceberg Quickstart
+
+This guide will get you up and running with an Iceberg and Hive environment, including sample code to
+highlight some powerful features. You can learn more about Iceberg's Hive runtime by checking out the [Hive](../docs/latest/hive/) section.
+
+- [Feature-support](#feature-support)
+- [Enabling Iceberg support in Hive](#enabling-iceberg-support-in-hive)
+- [Catalog Management](#catalog-management)
+- [DDL Commands](#ddl-commands)
+- [DML Commands](#dml-commands)
+
+### Docker images
+
+The fastest way to get started is to use [Apache Hive images](https://hub.docker.com/r/apache/hive) 
+which provides a SQL-like interface to create and query Iceberg tables from your laptop. You need to install the [Docker Desktop](https://www.docker.com/products/docker-desktop/), choosing the Intel or Apple M1 chip if you have a Mac, or choosing Linux.
+
+Set the version variable:
+```export HIVE_VERSION=4.0.0-alpha-2```
```

Comment:
? I think maybe we should reference how to discover the correct latest Hive version?

Reply:
https://hub.docker.com/r/apache/hive/tags?page=1&ordering=name

This is the best place from where we get all the images, just sorted desc.

Reply:
![image](https://github.com/apache/iceberg-docs/assets/16125343/ef7bd972-b588-4823-a14f-169cb664c94c)
Take a look at the Tags tab in [Apache Hive docker images](https://hub.docker.com/r/apache/hive/tags?page=1&ordering=-last_updated) to see the available Hive versions.